apps/hash-graph/lib/graph/src/api/gen.rs

```diff
@@ -0,0 +1,5 @@
+#![allow(dead_code)]
+
+pub mod status_payloads {
+    include!(concat!(env!("OUT_DIR"), "/status_payloads.rs"));
+}
```

Comment:
Is there a reason you don't add the `status.rs` module as well?

Reply:
So I assume if the Graph would _consume_ errors, it would be useful?

Reply:
No, I think it would never be useful for Rust. We could add an `exclude` pattern for the Rust generation but that is more work than is worth it at this stage considering the files are harmless.

The local `status` package has a crate with a natively implemented nicer generic `Status` type (which is what we import and use), and I think that would always be better than the generated one. We'd just need to define an alias of it which fills in the generic with the appropriate imported payload enum of whatever service is being _consumed_.